communities/microgalaxy/lab/sections/2_microbial_isolates.yml

```diff
@@ -1,9 +1,382 @@
+            GATK, 
+            Bcftools, 
+            Snippy, 
+            medaka, 
+            Clair3 
+
+            (16) VCFtools. SnpSift, 
+            SnpEff, 
+            Samtools 
+
+            (17) STAR, 
+            HISAT, 
+            Bowtie2 
+
+            (18) featureCounts, 
+            HTSeq 
+
+            (19) edgeR, 
+            DESeq2 
+
+            (20) StringTie, 
+            GFFCompare, 
+            Cufflints 
+            
+            (21) <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/msconvert/msconvert" target="_blank">msconvert</a>, 
+            OpenMS  (e.g. <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/openms_xtandemadapter/XTandemAdapter" target="_blank"> XTandemAdapter</a>)
+
+            (22) <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/search_gui" target="_blank">SearchGUI</a>,
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/peptideshaker/peptide_shaker" target="_blank">PeptideShaker</a>,
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/maxquant" target="_blank">MaxQuant</a>,
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/fragpipe/fragpipe" target="_blank">FragPipe</a>
+
+            (23) <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/maxquant/maxquant" target="_blank">MaxQuant</a>,
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/pepquery2/pepquery2" target="_blank">PepQuery</a>,
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/fragpipe/fragpipe" target="_blank">FragPipe</a>,
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/msstatstmt/msstatstmt" target="_blank">MSstatsTMT</a>,
+            iTRAQ,
+            <a href="{{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/unipept/unipept" target="_blank">UniPept</a>  
+
+            (24) MSStats  
+
+            <br><br>
+            <strong>Note:</strong> If a tool link does not work or is missing, it may not be installed on this server. Please contact 
+            <a href="{{ support }}">support</a> for assistance.  
+            If the tool is not listed here and you cannot find it in the tool's panel, you can request it via the microGalaxy community:  
+            <a href="mailto:microgalaxy@lists.galaxyproject.org">microgalaxy@lists.galaxyproject.org</a>.
+          </p>
+        </div>
+
+- id: tools
+  title: Tools
+  heading_md: >
+    Flagship tools for microbiome analysis
+  content:
+    subsections:
+      - id: genomics
+        title: Genomics
+        content:
+        - title_md: <code>MetaPhlAn</code> - A tool to profile the composition of microbial communities
```

Comment:
I would not put MetaPhlAn here. Probably Bakta is better. Can you add AMRFinderPlus? PPanGGoLin? antiSMASH?